Edit developer doc to use multiOptionalFactorEnroll in recovery flow (#3165)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/authn/index.md

| Parameter     | Description                                                                                                    | Param Type | DataType | Required |
| ------------- | ----------------------------------------------------------------------------------------------------------     | ---------- | -------- | -------- |
| recoveryToken | [Recovery token](#recovery-token) that was distributed to the end user via out-of-band mechanism such as email | Body       | String   | TRUE     |
| options                                        | Opt-in features for the authentication transaction                                                               | Body       | [Options object](#options-object) | FALSE    |    

##### Options object

> The authentication transaction [state machine](#transaction-state) can be modified via the following opt-in features:

Reviewer: ```suggestion
You can modify the authentication transaction [state machine](#transaction-state) through the following opt-in features:
```
